src/setup/devnet/templates/docker/docker-bor-config.toml.njk

```diff
@@ -0,0 +1,193 @@
+chain = "/var/lib/bor/genesis.json"
+identity = "MyIdentity"
+verbosity = 3
+log-level = ""
+vmdebug = false
+datadir = "/var/lib/bor/data"
+ancient = ""
+"db.engine" = "leveldb"
+keystore = "/var/lib/bor/keystore"
+"rpc.batchlimit" = 100
+"rpc.returndatalimit" = 100000
+syncmode = "full"
+gcmode = "full"
+snapshot = true
+"bor.logs" = true
+ethstats = ""
+devfakeauthor = false
+
+["eth.requiredblocks"]
+
+[log]
+  vmodule = ""
+  json = false
+  backtrace = ""
+  debug = false
+
+[p2p]
+  maxpeers = 200
+  maxpendpeers = 50
+  bind = "0.0.0.0"
+  port = 30303
+  nodiscover = false
+  nat = "any"
+  netrestrict = ""
+  nodekey = ""
+  nodekeyhex = ""
+  txarrivalwait = "500ms"
+  [p2p.discovery]
+    v5disc = false
+    bootnodes = []
+    bootnodesv4 = []
+    bootnodesv5 = []
+    static-nodes = []
+    trusted-nodes = []
+    dns = []
+
+[heimdall]
+  url = "http://localhost:1317"
+  "bor.without" = false
+  grpc-address = ""
+  "bor.runheimdall" = false
+  "bor.runheimdallargs" = ""
+  "bor.useheimdallapp" = false
+
+[txpool]
+  locals = []
+  nolocals = true
+  journal = "transactions.rlp"
+  rejournal = "1h0m0s"
+  pricelimit = 1
+  pricebump = 10
+  accountslots = 128
+  globalslots = 20000
+  accountqueue = 16
+  globalqueue = 32768
+  lifetime = "0h16m0s"
+
+[miner]
+  mine = true
+  etherbase = ""
+  extradata = ""
+  gaslimit = 20000000
+  gasprice = "1000000000"
+  recommit = "2m5s"
+  commitinterrupt = true
+
+[jsonrpc]
+  ipcdisable = false
+  ipcpath = "/var/lib/bor/data/bor.ipc"
+  gascap = 50000000
+  evmtimeout = "5s"
+  txfeecap = 5.0
+  allow-unprotected-txs = false
+  enabledeprecatedpersonal = false
+  [jsonrpc.http]
+    enabled = true
+    port = 8545
+    prefix = ""
+    host = "0.0.0.0"
+    api = ["eth", "net", "web3", "txpool", "bor"]
```

Comment:
is there anything else we want to enable by default in a devnet? Thinking in the `debug` direction